addons/main/XEH_preInit.sqf

```diff
@@ -140,124 +35,20 @@ ACRE_DUMPSTACK_FNC = {
     };
 };
 
-EFUNC(lib,getCompartment) = {
-    params ["_unit"];
-    private _vehicle = (vehicle _unit);
-    private _compartment = "";
-    if (_vehicle != _unit) then {
-        private _defaultCompartment = "Compartment1";
-        private _cfg = configFile >> "CfgVehicles" >> typeOf _vehicle;
-        private _assignedRole = assignedVehicleRole _unit;
-        if ((_assignedRole select 0) == "Driver") then {
-            _compartment = getText(_cfg >> "driverCompartments");
-            if (_compartment == "") then {
-                _compartment = _defaultCompartment;
-            };
-        } else {
-            if ((_assignedRole select 0) == "Turret") then {
-                private _turretPath = _assignedRole select 1;
-                private _turret = [_vehicle, _turretPath] call CBA_fnc_getTurret;
-                _compartment = getText(_turret >> "gunnerCompartments");
-                if (_compartment == "") then {
-                    _compartment = getText(_cfg >> "driverCompartments");
-                    if (_compartment == "") then {
-                        _compartment = _defaultCompartment;
-                    };
-                };
-            } else {
-                if ((_assignedRole select 0) == "Cargo") then {
-                    private _cargoCompartments = getArray(_cfg >> "cargoCompartments");
-                    if ((count _cargoCompartments) > 0) then {
-                        private _index = -1;
-                        // if ((productVersion select 3) < 126064) then {
-                            // _index = (count _attenuateCargo)-1; // wait for command to get cargo index
-                        // } else {
-                            _index = _vehicle getCargoIndex _unit;
-                        // };
-                        if (_index > -1) then {
-                            if (_index > (count _cargoCompartments)-1) then {
-                                _index = (count _cargoCompartments)-1;
-                            };
-
-                            _compartment = _cargoCompartments select _index;
-                        } else {
-                            _compartment = _defaultCompartment;
-                        };
-                    } else {
-                        _compartment = _defaultCompartment;
-                    };
-                };
-            };
-        };
-    };
-    if (!(typeName _compartment == "STRING")) then {
-        _compartment = _defaultCompartment;
-    };
-    _compartment;
-};
-
-EFUNC(lib,fastHashCreate) = {
-    if (count FAST_HASH_POOL > 0) exitWith {
-        private _ret = (FAST_HASH_POOL deleteAt 0);
-        FAST_HASH_CREATED_HASHES_NEW pushBack _ret;
-        _ret;
-    };
-    private _ret = createLocation ["AcreHashType", [-10000,-10000,-10000], 0, 0];
-    _ret setText "acre_hash";
-    FAST_HASH_CREATED_HASHES_NEW pushBack _ret;
-    _ret;
+EFUNC(sys_core,fastHashCreate) = {
```

Comment:
I'd suggest just adding a comment above this to explain the potential crash if moved.